interp/cfg.go

```diff
@@ -66,6 +66,24 @@ func (interp *Interpreter) cfg(root *node, importPath string) ([]*node, error) {
 			return false
 		}
 		switch n.kind {
+		case binaryExpr, unaryExpr, parenExpr:
```

Comment:
I understand why we need to catch all binary and paren expressions (and not just binary shift expressions), but I don't get why we need to catch unary expressions too. Can you give me an example?

Reply:
- (a * 2)

Reply:
although this crashes right now, maybe another issue

Reply:
I'm not sure what you meant, so out of curiosity I've tried:

```
package main

func main() {
    var max, a uint32
    max = 1 << - (a * 2)
    println(max) 
}
```

and `yaegi run` on that does not print anything. Maybe it's a different/another problem though, I haven't investigated further yet.

Reply:
ok, let me know if you want to merge this PR as is and investigate that independently. Otherwise I'll wait for more updates on this PR.